gdsfactory/pdk.py

```diff
@@ -240,7 +212,6 @@ class Pdk(BaseModel):
         routing_strategies: functions enabled to route.
         gds_write_settings: to write GDSII files.
         oasis_settings: to write OASIS files.
-        cell_decorator_settings: settings for the cell decorator.
```

Comment:
Removing this doesn't seem backwards-compatible, but I've never personally used this feature.

Reply:
I brought this back with less parameters